application/common/models/Authentication.php

```diff
@@ -74,9 +77,15 @@ protected function authenticateByInvite($invite)
             $this->errors['invite'] = ['Invalid code.'];
             return;
         }
+
         if ( ! $invite->isValidCode()) {
```

Comment:
it looks like an invalid code that is NOT expired will generate a new code and email, am I reading that right?

Reply:
I was afraid of that being misleading. The previous `if` ensures that the code is a real code. `isValidCode` only checks expiration.

Reply:
would changing the name to `isExpired` avoid the confusion?

Reply:
done